src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int SEARCH_BY_INDEX = 0;
+    private static final int COUNT_BY_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFile = new File(fromFileName);
+        File writeFile = new File(toFileName);
+        try {
+            readingFile(readFile, writeFile);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't read data from file", e);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
+        }
+    }
+
+    private void readingFile(File readFile, File writeFile) throws IOException {
```

Comment:
```suggestion
    private void readingFile(File readFile, File writeFile) {
```
Instead of throwing the error higher up, catch it where it arises